notion/tool.gpt

```diff
@@ -62,12 +73,16 @@ Name: Update Database Row
 Description: Updates an existing row in a database in Notion. Call Get Database Properties prior to this to get the properties of the database.
 Credential: github.com/gptscript-ai/gateway-oauth2 as notion.write with NOTION_TOKEN as env and notion as integration
 Share Context: addDatabaseRowInstructions from context.gpt
+Share Tools: Search
```

Comment:
This one probably also needs `Get Database`, so that it knows which rows there are.